common/beerocks/bcl/include/bcl/beerocks_socket_thread.h

```diff
@@ -42,6 +42,7 @@ class socket_thread : public thread_base {
     {
         return false;
     }
+    virtual bool skip_filtered_message_type(ieee1905_1::eMessageType msg_type) { return false; }
```

Comment:
I don't like this name... Maybe change it to `should_handle_ieee1905_msg()`?

Reply:
better